lang/clang-11-bootstrap/Portfile

```diff
@@ -0,0 +1,275 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           muniversal 1.0
+
+# clang-11 is the last clang which can be build with the last version of cmake to not require c++11.
+name                clang-11-bootstrap
+
+version             11.1.0
+revision            0
+epoch               0
+
+platforms           darwin
+categories          lang
+maintainers         {@catap korins.ky:kirill} openmaintainer
+
+license             NCSA
+
+description         clang-11 with minimal MacPorts patches pegged at version ${version}.
+long_description    ${description} This port depends only on bootstrap dependencies.
+
+homepage            https://llvm.org/
+
+
+master_sites        https://github.com/llvm/llvm-project/releases/download/llvmorg-${version}
+distname            llvm-project-${version}.src
+
+checksums           rmd160  f566b4b75c8f30418f19069a9a84864ead766401 \
+                    sha256  74d2529159fd118c3eac6f90107b5611bccc6f647fdea104024183e8d5e25831 \
+                    size    84065492
+
+# no Clang / LLVM for PowerPC yet
+supported_archs     arm64 i386 x86_64
+
+depends_extract     port:xz-bootstrap
+depends_skip_archcheck-append \
+                    xz-bootstrap
+
+extract.suffix      .tar.xz
+extract.cmd         ${prefix}/libexec/xz-bootstrap/bin/xz
+
+depends_build-append \
+                    port:python27-bootstrap
+depends_skip_archcheck-append \
+                    python27-bootstrap
+
+# gcc10-bootstrap is used as compiler and as libc++
+# to avoid such dependency this port should also build libc++
+# and use 2-stage bootstrap
+depends_build-append \
+                    port:gcc10-bootstrap
+depends_skip_archcheck-append \
```

Comment:
Remove this as well want gcc10-bootstrap arch to match now that universal seems to be correctly fixed.